src/hooks/use-load-topic-data.ts

```diff
@@ -1,26 +1,74 @@
 import * as React from 'react'
 import groq from 'groq'
 import {sanityClient} from '@/utils/sanity-client'
+import {topicExtractor} from '@/utils/search/topic-extractor'
+import {first} from 'lodash'
+import {loadTag} from '@/lib/tags'
+
+function useLoadTopicData(
+  initialTopicGraphqlData: any,
+  initialTopicSanityData: any,
+  searchState: any,
+) {
```

Comment:
I merged the two loader hooks into this one so that I could have more control of the timing